--- /etc/amavisd.conf
+++ /etc/amavisd.conf
@@ -10,7 +10,7 @@
 # COMMONLY ADJUSTED SETTINGS:
 
 # @bypass_virus_checks_maps = (1);  # controls running of anti-virus code
-# @bypass_spam_checks_maps  = (1);  # controls running of anti-spam code
+@bypass_spam_checks_maps  = (1);  # controls running of anti-spam code
 # $bypass_decode_parts = 1;         # controls running of decoders&dearchivers
 
 # $myprogram_name = $0; # set to 'amavisd' or similar to avoid process name
@@ -20,7 +20,8 @@
 $daemon_user   = 'amavis';   # (no default;  customary: vscan or amavis), -u
 @daemon_groups  = 'amavis';  # (no default;  customary: vscan or amavis), -g
 
-$mydomain = 'example.com';   # a convenient default for other settings
+$mydomain = "$ENV{DOMAIN}";   # a convenient default for other settings
+$inet_socket_bind = '0.0.0.0';
 
 $MYHOME = '/var/amavis';
 $TEMPBASE = "$MYHOME/tmp";
@@ -38,7 +39,7 @@
 # $pid_file  = "$MYHOME/var/amavisd.pid";   # -P
 #NOTE: create directories $MYHOME/tmp, $MYHOME/var, $MYHOME/db manually
 
-$log_level = 0;              # verbosity 0..5, -d
+$log_level = 2;              # verbosity 0..5, -d
 $log_recip_templ = undef;    # disable by-recipient level-0 log entries
 $do_syslog = 1;              # log via syslogd (preferred)
 $syslog_facility = 'mail';   # Syslog facility as a string
@@ -48,14 +49,16 @@
 $enable_db = 0;              # enable use of BerkeleyDB/libdb (SNMP and nanny)
 # $enable_zmq = 1;           # enable use of ZeroMQ (SNMP and nanny)
 $nanny_details_level = 2;    # nanny verbosity: 1: traditional, 2: detailed
-$enable_dkim_verification = 1;  # enable DKIM signatures verification
-$enable_dkim_signing = 1;    # load DKIM signing code, keys defined by dkim_key
+$enable_dkim_verification = 0;  # enable DKIM signatures verification
+$enable_dkim_signing = 0;    # load DKIM signing code, keys defined by dkim_key
 
 @local_domains_maps = ( [".$mydomain"] );  # list of all local domains
 
 @mynetworks = qw( 127.0.0.0/8 [::1] [FE80::]/10 [FEC0::]/10
                   10.0.0.0/8 172.16.0.0/12 192.168.0.0/16 );
 
+@inet_acl = qw( 0/0 [::] );
+
 $unix_socketname = "$MYHOME/amavisd.sock";  # amavisd-release or amavis-milter
                # option(s) -p overrides $inet_socket_port and $unix_socketname
 
@@ -165,10 +168,10 @@
 
 # OTHER MORE COMMON SETTINGS (defaults may suffice):
 
-# $myhostname = 'host.example.com';  # must be a fully-qualified domain name!
+$myhostname = "$ENV{MYHOSTNAME}";  # must be a fully-qualified domain name!
 
-# $notify_method  = 'smtp:[127.0.0.1]:10025';
-# $forward_method = 'smtp:[127.0.0.1]:10025';  # set to undef with milter!
+$notify_method  = "smtp:[$ENV{SMTPHOST}]:$ENV{SMTPPORT}";
+$forward_method = "smtp:[$ENV{SMTPHOST}]:$ENV{SMTPPORT}";  # set to undef with milter!
 
 # $final_virus_destiny      = D_DISCARD;
 # $final_banned_destiny     = D_DISCARD;
@@ -813,10 +816,10 @@
 # socket to match your system. The path is usually defined in the file
 # /etc/clamd.conf, or may be controlled by your service manager / init.
 #
-# ['ClamAV-clamd',
-#   \&ask_daemon, ["CONTSCAN {}\n", "/var/run/clamav/clamd.sock"],
-#   qr/\bOK$/m, qr/\bFOUND$/m,
-#   qr/^.*?: (?!Infected Archive)(.*) FOUND$/m ],
+ ['ClamAV-clamd',
+   \&ask_daemon, ["CONTSCAN {}\n", "clamd:3310"],
+   qr/\bOK$/m, qr/\bFOUND$/m,
+   qr/^.*?: (?!Infected Archive)(.*) FOUND$/m ],
 
 # ['ClamAV-clamd-stream',
 #   \&ask_daemon, ["*", 'clamd:/var/run/clamav/clamd.sock'],
